src/routes/Categories.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { checkStatus } from '../redux/categories/categoriesSlice';
 
 export default function Categories() {
+  const dispatch = useDispatch();
+  const status = useSelector((state) => state.categories.status);
   return (
     <div className="category-wrapper">
-      <button type="button" className="check-status-btn">Check status </button>
+      <p className="status-display">{status}</p>
+      <button type="button" className="check-status-btn" onClick={() => dispatch(checkStatus())}>Check status </button>
     </div>
   );
 }
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you move the `src/routes/Categories.js` file in the `src/components/` directory for good files structure.